requirements/dev-requirements.txt

```diff
@@ -794,15 +793,14 @@ turn-python==0.0.1
     # via -r base-requirements.in
 twilio==7.12.0
     # via -r base-requirements.in
-typing-extensions==4.7.1
+typing-extensions==4.1.1
```

Comment:
This reverts another requirements version changed in the previous PR - I believe brought about by the sqlalchemy change